NEWS

```diff
@@ -180,7 +204,7 @@ v5.0.0rc2 -- October, 2021
   for the contribution.
 - fcoll/two_phase: Removed the component. All scenerios it was
   used for has been replaced.
-- btl/uct: Bumped UCX allowed version to v1.9.x.
+- btl/uct: Bumped UCX allowed version to v1.10.x.
```

Comment:
Is this v1.10.x the oldest or the newest UCX allowed version?  (I assume the oldest)?

Reply:
I think the ceiling.

Reply:
https://github.com/open-mpi/ompi/commit/d1ea10520617831057f351c0ce1f2fd09f6599b6